server/models/user.js

```diff
@@ -1 +1,71 @@
 // init
+const mongoose = require('mongoose')
+const jwt = require('jsonwebtoken')
+
+const userSchema = new mongoose.Schema({
+    un : {  // username

```

Comment:
A good thing to keep in mind is that a variable name should not need a comment to explain its purpose. So if you had named it `username` instead of `un` you wouldn't have needed the `// username` comment.